DemoApp/DemoApp/DataAccessLayer/Services/StudentsDataService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using BusinessLayer.Models;
+using DataAccessLayer.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace DataAccessLayer.Services
+{
+    public class StudentsDataService : IStudentsDataService
+    {
+        private readonly SchoolContext _context;
+
+        public StudentsDataService(SchoolContext context) 
+        {
+            _context = context;
+        }
+
+        public async Task<IEnumerable<Student>> GetStudentsAsync()
+        {
+            return await _context.Students.ToListAsync();
```

Comment:
Why IEnumerable ?
You return a list right?

Reply:
cahnged to List